h2/src/main/org/h2/fulltext/FullTextLucene.java

```diff
@@ -312,6 +318,17 @@ protected static IndexAccess getIndexAccess(Connection conn)
         }
     }
 
+    private static Analyzer createAnalyzer() {
+        try {
+            return (Analyzer) Class.forName(Utils.getProperty(LUCENE_ANALYZER,
+                    "org.apache.lucene.analysis.standard.StandardAnalyzer"))
+                    .getConstructor(Version.class)
+                    .newInstance(Version.LUCENE_30);
```

Comment:
StandardAnalyzer constructor has no parameters in newer versions

Reply:
Got it, will retrofit it.